### ISSUE TITLE
<kbd> elements render on its own line in .epub files

### ISSUE THREAD

**Issue by chrisr87** [max-heller/mdbook-pandoc#110]
I may be missing an option that I'm supposed to pass here to get `<kbd>` elements to correctly render inline. Thoughts?

```
# book.toml
[book]
title = "The Rust Programming Language"
authors = ["Steve Klabnik", "Carol Nichols", "Contributions from the Rust Community"]

[rust]
edition = "2021"

[output.pandoc]
hosted-html = "https://doc.rust-lang.org/book"

[output.pandoc.profile.epub]
output-file = "rust-book.epub"
table-of-contents = false

[output.pandoc.profile.epub.variables]
epub-cover-image = "cover.png"
epub-title-page = false
```

**Comment by max-heller** [comment]
This seems to be the result of a preprocessing pass that [inserts divs to make sure Pandoc generates valid EPUB](https://github.com/max-heller/mdbook-pandoc/blob/main/src/lib.rs#L1227-L1282). It shouldn't be touching inline HTML like this, so I'll look into a fix

**Comment by max-heller** [comment]
#112 should fix this issue, thanks for reporting! I'll publish a new release shortly

**Comment by max-heller** [comment]
I [released 0.7.2](https://github.com/max-heller/mdbook-pandoc/releases/tag/v0.7.2) which should fix this
